### PR TITLE
Change Swift to experimental

### PR DESCRIPTION
**Pull request by nmote**
As of https://github.com/returntocorp/semgrep/pull/5870, Swift meets the bar for experimental status!

# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs

### CODE REVIEW

**Review by adamkvitek — APPROVED** [review]

**Comment by adamkvitek** [comment]
@nmote thank you! Is this already available now or are we waiting for some release? Feel free to merge this PR when this support is available for users.

**Comment by nmote** [comment]
Yeah, this won't be true until next week's Semgrep release, which is why I had it marked as a draft. I will merge it once that's out!

**Pull request merged**
